### PR TITLE
[ligthouse] add "descriptive link text" reference

### PR DESCRIPTION
**Pull request by kaycebasques**
greetings @rviscomi ... got another one for ya

What's changed, or what was fixed?
- add reference for "links do not have descriptive text" audit

**Fixes:** N/A

**Target Live Date:** 2018-02-28

- [x] This has been reviewed and approved by @rviscomi 
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by petele** [comment]
@rviscomi - if you approve, please feel free to "Squash and Merge"

File: src/content/en/tools/lighthouse/audits/descriptive-link-text.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    <p>To see all of our basketball videos, <a href="videos.html">click here</a>.</p>
+
+With specific descriptions, such as `basketball videos` in the example below.
+
+    <p>Check out all of our <a href="videos.html">basketball videos</a>.</p>
+
+In general, the link text should clearly indicate to users what type of content they'll get if
+they click the link. Using the exact title of the document, or a description of the page, is
+usually good enough. Avoid using URLs as link descriptions.
```
Comment (rviscomi) [inline src/content/en/tools/lighthouse/audits/descriptive-link-text.md]: s/link/anchor/ for consistency with the overview above?

File: src/content/en/tools/lighthouse/audits/descriptive-link-text.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+[style]: https://developers.google.com/style/link-text
```
Comment (rviscomi) [inline src/content/en/tools/lighthouse/audits/descriptive-link-text.md]: Is this used anywhere?

File: src/content/en/tools/lighthouse/audits/descriptive-link-text.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Links Do Not Have Descriptive Text" Lighthouse audit.
+
+{# wf_updated_on: 2018-02-22 #}
+{# wf_published_on: 2018-02-22 #}
+{# wf_blink_components: N/A #}
+
+# Links Do Not Have Descriptive Text  {: .page-title }
```
Comment (rviscomi) [inline src/content/en/tools/lighthouse/audits/descriptive-link-text.md]: How much do you think these docs should go into the SEO benefits of [doing the thing the audit checks for]? Linking to relevant SEO docs is good too.

File: src/content/en/tools/lighthouse/audits/descriptive-link-text.md (hunk 1 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/descriptive-link-text.md]: Good catch, I thought about this too, but opted to use both terms to signify that they are interchangeable, and improve our own SEO

File: src/content/en/tools/lighthouse/audits/descriptive-link-text.md (hunk 2 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/descriptive-link-text.md]: No, good catch, thanks

File: src/content/en/tools/lighthouse/audits/descriptive-link-text.md (hunk 3 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/descriptive-link-text.md]: I thought about that, but I haven't really seen anything else on it, other than "it helps search engines understand your content"

We could link to webmaster blog, to give it authority, but the content is pretty much duplicated

Comment (rviscomi) [reply src/content/en/tools/lighthouse/audits/descriptive-link-text.md]: Here's one: https://support.google.com/webmasters/answer/7451184#optimize (scroll down to "Write good link text" section, no closer anchor to jump to... grr)

**Pull request merged**